packages/utils/scss/layout/_position.scss

```
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
     left: 50%;
     transform: translateX(-50%);
 }
+
+
+/// @name k-pos-top
+/// @description This is equivalent to `top: 0;`.
+/// @group position
+
+/// @name k-pos-right
+/// @description This is equivalent to `right: 0;`.
+/// @group position
+
+/// @name k-pos-bottom
+/// @description This is equivalent to `bottom: 0;`.
+/// @group position
+
+/// @name k-pos-left
+/// @description This is equivalent to `left: 0;`.
+/// @group position
+
+@each $side in (top, right, bottom, left) {
+    .k-#{$side},
+    .k-pos-#{$side} { #{$side}: 0; } // sass-lint:disable-line brace-style
+}
+
+/// @name k-pos-top-start
+/// @description This is equivalent to `top: 0; left: 0;`.
+/// @group position
+.k-top-start,
+.k-pos-top-start {
+    top: 0;
+    left: 0;
+
+    .k-rtl &,
+    &[dir="rtl"],
+    [dir="rtl"] & {
+        left: auto;
+        right: 0;
+    }
```

you can remove the rtl parts:

```suggestion
.#{$kendo-prefix}top-start,
.#{$kendo-prefix}pos-top-start {
    inset-block-start: 0;
    inset-inline-start: 0;
}
```